databricks/koalas/indexes.py

```diff
@@ -472,6 +475,54 @@ def any(self, *args, **kwargs):
     def all(self, *args, **kwargs):
         raise TypeError("cannot perform all with this index type: MultiIndex")
 
+    @classmethod
```

Comment:
`@staticmethod` is enough?

Reply:
oh i just researched the more about their difference, and it's better to use `@staticmethod` as you said. thanks for the comment!!